faqs/galaxy/histories_finding_histories.md

```diff
@@ -0,0 +1,14 @@
+title: Finding Histories
+area: histories
+box_type: tip
+layout: faq
+contributors: [Melkeb]
```

Comment:
don't forget to add yourself to the CONTRIBUTORS.yaml file for this to work :)

If we can ascertain who wrote the original FAQ on the hub, it would be great to add them here as well

Reply:
It's probably safe to add @jennaj to all of these:

```suggestion
contributors: [jennaj, Melkeb]
```

(And if you know others who contributed there @jennaj please add them too)